### PR TITLE
Companions table 

### PR DESCRIPTION
**Pull request by Ajb2307**
- [x] add ingest companions function 
- [x] add other names ingest function 
- [x] write tests for companions
- [x] write tests for other names 
- [x] check that tests work
- [x] moves tests to test utils 
- [x] add other companion names
- [x] tests for companion relationships

### CODE REVIEW

**Comment by Ajb2307** [comment]
Note because there's no companion relationship schema this branch will fail

**Comment by kelle** [comment]
Go ahead and commit your modifications to schema.py.  Your changes supersede what I started anyway.

**Comment by kelle** [comment]
I added the documentation file here so I'll close my PR in favor of this one.

File: tests/test_utils.py [hunk 1]
```diff
@@ -261,3 +261,41 @@ def test_ingest_instrument(db):
...
+def test_companion_relationships(db):
+# testing companion ingest
+## trying no companion 
+    tests_passed = 0
+    try:
+        ingest_companion_relationship(db, 'Fake 1', None )
+    except SimpleError as e:
+        print('Companionless source in CompanionRelationship BLOCKED (as expected)')
+        tests_passed += 1
+        pass
```
Comment (kelle) [inline tests/test_utils.py:274]:
```suggestion
    with pytest.raises(SimpleError) as error_message:
        ingest_companion_relationship(db, 'Fake 1', None )
        assert 'some string' in str(error_message.value)
```

**Review by kelle — CHANGES_REQUESTED** [review]
The tests in `test_utils` should be setup differently using `pytest.raises` and not try/excepts.

File: tests/test_integrity.py [hunk 2]
```diff
@@ -557,3 +571,91 @@ def test_remove_database(db):
     db.engine.dispose()
```
Comment (dr-rodriguez) [inline tests/test_integrity.py:571]: The `test_remove_database` function deletes the database. There should be no tests after this.

**Review by kelle — APPROVED** [review]
LGTM!

**Pull request merged**